pkg/kubelet/dockertools/manager.go

```diff
@@ -140,6 +142,21 @@ type DockerManager struct {
 	enableCustomMetrics bool
 }
 
+// Option is a functional option type for DockerManager, useful for
+// completely optional settings.
+type Option func(*DockerManager)
```

Comment:
nit: Can we pull this out of docker repo and place it in `pkg/kubelet/container` package? The goal was to make it cross-runtime compatible.